source/templates.tex

```diff
@@ -5881,11 +5884,34 @@
 
 \rSec2[temp.inst]{Implicit instantiation}
 
+\pnum
+A template specialization $E$ is a \defnadj{declared}{specialization}
+if there is a reachable
+explicit instantiation definition\iref{temp.explicit} or
+explicit specialization declaration\iref{temp.expl.spec}
+for $E$, or
+if there is a reachable explicit instantiation declaration for $E$ and
+$E$ is not
+\begin{itemize}
+\item
+an inline function,
+\item
+declared with a type deduced
+from its initializer or return value\iref{dcl.spec.auto},
+\item
+a potentially-constant variable\iref{expr.const}, or
+\item
+a specialization of a templated class.
```

Comment:
What is a "templated class"?

Reply:
A templated entity that is a class. (Any fixed in this area should be separate.)

Reply:
Please replace "templated class" editorially in a separate commit with: "templated entity that is a class type" (since at least "templated entity" is defined).

Reply:
Sorry, won't do. CWG is contemplating changing the definition to "An entity is templated if..." , which might even be an editorial fix.